mungegithub/submit-queue/www/script.js

```diff
@@ -235,15 +238,25 @@ function SQCntl(dataService, $interval, $location) {
           failedBuild = true;
       }
       obj.stability = '';
-      result.push(obj);
+      if (!obj.msg) {
+        result.push(obj);
+      } else {
+        if (obj.msg.includes('[nonblocking]')) {
+          obj.msg = ''
```

Comment:
I suggest not clearing it, just remove the '[nonblocking]' prefix since the rest of the message may say something useful.